code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/views.cljs

```diff
@@ -300,9 +296,19 @@
         deploy-price   (str (@tr [(if follow-trial?
                                     :free-trial-and-then
                                     :deploy-for)])
-                         (format-money (/ (:cent-amount-daily price) 100)) "/"
-                         (@tr [:day]))
+                            (format-money (/ (:cent-amount-daily price) 100)) "/"
+                            (@tr [:day]))
         button-content "Add to selection"
+        button-ops     {:fluid   true
```

Comment:
cleanup button-ops redefined. Fix the merge conflict and you can merge it after that 👍